recipes/libvpx/all/conanfile.py

```diff
@@ -1,20 +1,27 @@
-from conan.tools.microsoft import msvc_runtime_flag
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os, fix_apple_shared_install_name
+from conan.tools.env import Environment, VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, replace_in_file, rename
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc, VCVars, unix_path, msvc_runtime_flag
 from conan.tools.microsoft.visual import msvc_version_to_vs_ide_version
```

Comment:
another case where https://github.com/conan-io/conan/issues/12322 would be helpful 

linter complains because it;s private

Reply:
![image](https://user-images.githubusercontent.com/16867443/214626710-cf3002da-b292-423d-a8b0-db0c4b073c93.png)

For sheppards

Reply:
```py
if is_msvc(self) and Version(self.settings.compiler.version) > "100":
    ide_version = {"190": "14", "191": "15", "192": "16"}
```

Reply:
As @prince-chrismc  currently the method is private and the linter is rightfully warning us about this. This is a good case to make it public, but if that were to be the case it would end up with a different import pattern.

@prince-chrismc 's suggestion above (or something similar) might be enough to unblock this. If it works satisfactorily, please mention the issue in a comment next to the code so that we can clean this up in the future! Thanks :)